mediadevices.go

```diff
@@ -218,7 +218,6 @@ func selectBestDriver(filter driver.FilterFn, constraints MediaTrackConstraints)
 	}
 
 	constraints.selectedMedia = bestProp
-	constraints.selectedMedia.Merge(constraints.MediaConstraints)
```

Comment:
I think it is needed for the parameters which can be given from continuous range of the value.
Currently the main usage is FrameRate of screen capture driver.

Reply:
Yep, that makes sense.